src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -1,9 +1,10 @@
 # Genetic Descriptor
 
-Support for genetic descriptors was developed as a [BIDS Extension
-Proposal](../07-extensions.md#bids-extension-proposals).
-The extension was primarily developed by Cyril Pernet and Clara Moreau with
-contributions from Tom Nichols and Jessica Turner.
+Support genetic descriptors was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
```

Comment:
This one isn't currently listed in the Citing BIDS section, AFAICT ?

Reply:
Agreed. Let me see if I can find the citation info. Unless @CPernet has it to hand?

Reply:
I think it will be something like:

> Pernet, C., Moreau, C., Jean-Louis, M., Blair, R., Markiewicz, C., Turner, J., Calhoun, V., Nichols, T. "The genetics-BIDS extension: Easing the search for genetic data associated with human brain imaging ". GigaScience (in press).

There's no DOI yet.

Reply:
indeed, I wanted to "set the scene" to just paste the info once @CPernet has it. See: https://github.com/bids-standard/bids-validator/issues/1065#issuecomment-692788943

But we can also put the "unfinalized" reference here.

Reply:
Ah, I knew there was a discussion somewhere...

Reply:
I added the WIP reference.